spec/system/comments/user_fills_out_comment_spec.rb

```diff
@@ -24,6 +25,20 @@
     expect(page).to have_text(raw_comment)
   end
 
+  it "User makes too many comments" do
```

Comment:
This is pretty much the way to test non-Preact/ES module code for now. Is the submit button being tested here, the one in the modal? Maybe check for some other things in the modal as well.